### DISCUSSION TITLE
Unable to enable regex manager

### DISCUSSION THREAD

**Discussion by YorickH** [renovatebot/renovate#19631] [Request Help] [answered]
In short, we have a yaml file that contains a bunch of different helm versions.

```
---

# renovate: depName=argo-rollouts-chart packageName=argo-rollouts registryUrl=https://argoproj.github.io/argo-helm
argo_rollouts_chart_version: 2.18.0
# renovate: depName=argocd-chart packageName=argo-cd registryUrl=https://argoproj.github.io/argo-helm
argocd_chart_version: 5.14.1
# renovate: depName=aws-load-balancer-controller-chart packageName=aws-load-balancer-controller registryUrl=https://aws.github.io/eks-charts
aws_load_balancer_controller_chart_version: 1.4.1
# renovate: depName=cert-manager-chart packageName=cert-manager registryUrl=https://charts.jetstack.io
cert_manager_chart_version: 1.8.0
```

I am trying to get this list into renovate by using the comments and a regex manager
I have the following configuration set:

```
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:base"
  ],
  "enabledManagers": ["regex"],
  "regexManagers": [
    {
      "fileMatch": ["(^|/)charts\\.yaml$"],
      "matchStrings": [ "depName=(?<depName>.+?) packageName=(?<packageName>.+?) registryUrl=(?<registryUrl>.+?)\\n+.+?: (?<currentValue>.*)" ],
      "datasourceTemplate": "helm"
    }
  ]
}

```

When looking at the logs i find these the most noteworthy: 
```
DEBUG: Applying enabledManagers filtering
DEBUG: Using file match: (^|/)charts\.yaml$ for manager regex
DEBUG: Matched 1 file(s) for manager regex: playbooks/roles/k8s_base/defaults/main/charts.yaml
DEBUG: Found 0 package file(s)
DEBUG: Manager explicitly enabled in "enabledManagers" config, but found no results. Possible config error?
{
  "manager": "regex"
}
INFO: Dependency extraction complete
{
  "baseBranch": "master",
  "stats": {
    "managers": {},
    "total": {
      "fileCount": 0,
      "depCount": 0
    }
  }
}
```
No other WARN, ERROR, or FATAL messages are to be found. 

Before, I did not enabled `enabledManagers` and had even less to go from. 
The `playbooks/roles/k8s_base/defaults/main/charts.yaml` path is the correct file where the above list is located.

I have tried the regex on regex101.com: https://regex101.com/r/wG2w6C/2

It's my first time playing around with the regex manager. We are using the GitHub app and this is happening on the initial configure renovate PR.
Am I missing something? I feel like I have made a dumb mistake, forgot something or I am misunderstanding something.

**Comment by viceice** [comment]
i think your regex is not matching

**Comment by YorickH** [comment, marked as answer]
Thanks for the suggestion, but shortly after writing my question I started testing with a minimal reproduction repository and there it worked.

The solution was that I was overlooking something, the yaml changes (renovate comments) were not merged into the main branch. 
See https://github.com/YorickH/renovate-reproduction/pull/3 for a working example.
